src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -710,6 +711,21 @@ public Entity get(LightningStrikeEvent event) {
 				return event.getLightning();
 			}
 		}, 0);
+		// EndermanAttackPlayerEvent
+		if (Skript.classExists("com.destroystokyo.paper.event.entity.EndermanAttackPlayerEvent")) {
+			EventValues.registerEventValue(EndermanAttackPlayerEvent.class, Entity.class, new Getter<Entity, EndermanAttackPlayerEvent>() {
+				@Override
+				public Entity get(EndermanAttackPlayerEvent event) {
+					return event.getEntity();
+				}
+			}, EventValues.TIME_NOW);
```

Comment:
This value shouldn't actually be needed as EndermanAttackPlayerEvent is a type of EntityEvent (for which event-entity is already registered).

```suggestion
```